Metered Websockify + integrated NoVNC (#749)
netgroup-polito/CrownLabs: provisioning/containers/gui-common/README.md

This container has been designed to be executed as a sidecar inside the same pod of the VNC server without any further configuration.
However, it could be executed in other contexts as well by setting the necessary environment variables, if needed.

> In CrownLabs, a custom implementation is used, derived from the go version available on [C version of Websockify](https://github.com/novnc/websockify-other) in order to provide a more efficient (in terms of space, memory and cpu usage) and customized component.

Reviewer: ```suggestion
In CrownLabs, a custom implementation is used, derived from the go version available in the [Websockify-other repository](https://github.com/novnc/websockify-other) in order to provide a more efficient (in terms of space, memory and cpu usage) and customized component.
```
